README.md

```diff
@@ -80,7 +80,14 @@ tcpstat | Exposes TCP connection status information from `/proc/net/tcp` and `/p
 Name     | Description | OS
 ---------|-------------|----
 gmond | Exposes statistics from Ganglia. | _any_
-megacli | Exposes RAID statistics from MegaCLI. | Linux
+
+### Removed
```

Comment:
Do you really need to mention these still?

Reply:
We never updated the README with a mention of the new textfile collector. I wanted there be some way of knowing where to find it.